build.gradle

```diff
@@ -509,6 +515,33 @@ configure(opentelemetryProjects) {
         // Always run jmhReport after jmh task.
         tasks.jmh.finalizedBy tasks.jmhReport
     }
+
+    // We vendor in a shaded version of weak-lock-free into opentelemetry-context, which is a
```

Comment:
@trask Please review the shading I've set up in this PR.

Reply:
Have you verified the .pom/.module and META-INF still look OK for this? IIRC shadow by default does not preserve the MANIFEST.mf of any patched jars.

Reply:
Thanks! Realized weak-lock-free was ending up in `opentelemetry-context` POM, changed so that doesn't happen.